homeassistant/components/netatmo/climate.py

```diff
@@ -155,113 +161,82 @@ def temperature_unit(self):
     @property
     def current_temperature(self):
         """Return the current temperature."""
-        return self._data.room_status[self._room_id]['current_temperature']
+        return self._current_temperature
 
     @property
     def target_temperature(self):
         """Return the temperature we try to reach."""
-        return self._data.room_status[self._room_id]['target_temperature']
+        return self._target_temperature
+
+    @property
+    def target_temperature_step(self) -> Optional[float]:
+        """Return the supported step of target temperature."""
+        return PRECISION_HALVES
 
     @property
     def hvac_mode(self):
-        """Return the current state of the thermostat."""
-        return self._operation_mode
+        """Return hvac operation ie. heat, cool mode."""
+        return self._hvac_mode
 
     @property
     def hvac_modes(self):
-        """Return the operation modes list."""
-        return self._operation_list
+        """Return the list of available hvac operation modes."""
+        return SUPPORT_HVAC
```

Comment:
Why do you maintain `self._operation_list` if you are not using it ?

Reply:
True, thanks for the hint.